renderdoc/driver/gl/gl_replay.cpp

```diff
@@ -254,32 +257,44 @@ void GLReplay::SetReplayData(GLWindowingData data)
     }
   }
 
-  if(isMesa)
+  if(isMesa && m_DriverInfo.vendor != GPUVendor::Intel)
```

Comment:
Correct, the Mesa driver needs to provide pretty close feature parity on the performance counter side because we need to support Intel GPA (https://software.intel.com/en-us/gpa)